pythonarcade/arcade
# Clarify compute shader N-body tutorial

### Changes

* Add optional star color to make movement visualization easier
* Rephrase tutorial page, replacing "ball" with "star" in Python & shader source to be consistent with the example theme
* Lower default resolution to accommodate smaller screens
* Lower default number of stars to accommodate weak GPUs
* Add top level constant + units in comments on existing top-level constants
* Reorganize code to follow tutorial's structure
* Clarify comments & docstrings
* Update Window title to match top level docstring

### Why

This addresses issues I found while reading the code to understand #1782.

### How to Test

1. Run `./make.py serve`
2. Go to http://localhost:8000/tutorials/compute_shader/index.html
3. Proof read for accuracy
4. Run `main.py` in the example folder to make sure nothing new is broken, only the movement issues previously reported

### Follow-up work

1. Attempt a fix for #1782
2. (TBD / uncertain) Compact this example to use a more efficient layout in memory?
3. (TBD/ uncertain) Separate into steps like other tutorials?
